### ISSUE TITLE
Status of this project?

### ISSUE THREAD

**Issue by sandrinr** [phusion/baseimage-docker#361]
Hi

I see that there was no update to this project for quite some time. At the same time there are some open (valid) pull requests and forks around. Also the latest build on Dockerhub is old.

@maintainers: What is the status of this project in your regard? Are you not using this baseimage anymore inside Phusion?

Regards,
Raffaele

**Comment by FooBarWidget** [comment]
We are still using this image actively, but we haven't felt the need for major updates so far, at least for our own use cases. If you are worried about security then there are clear instructions in the README on how to apply the latest security updates yourself. Because of these factors, maintenance of this image has been put on a lower priority in favor of other things that are happening in the organization (we aren't making money from this image, so we can only dedicate free time to the maintenance of this image, which unfortunately is not much).

I believe that it may be time to recruit additional maintainers for this project so that they can help serving the community interest. If anybody would like to step up, please let me know.

**Comment by jarischaefer** [comment]
I think basing the image on Ubuntu's LTS releases is sufficient to be up to date. However, many packages receive updates during the two years until the next LTS is released. Therefore, the image's size also grows when updating.

Release 0.9.19 is 226MB. A fully updated image based on 0.9.19 is roughly 100MB larger. Perhaps it would be sufficient to introduce a "static" 0.9.19 that is built only once (when the LTS is released) and a 0.9.19-current that is built regularly and contains the latest updates (still based on the LTS). But this is mostly nice to have, I don't care _that_ much about 100MB, but that may be something other people would like to see (think of those with slow up-/dow speeds).

In regard to the pull requests, I agree that additional maintainers might help tackle these.

**Comment by optimisticanshul** [comment]
@FooBarWidget I would love to maintain it, if you can add me as maintainer.

**Comment by FooBarWidget** [comment]
Hello everybody, I'm pleased to announce that @theaxiom has been appointed as the new maintainer! We've been discussing this for a while now and today I've finalized the process.

@optimisticanshul thank you for volunteering as well. @Theaxiom was just slightly faster, but he has indicated interest in sharing maintainership. You two should talk. :)

**Comment by optimisticanshul** [comment]
@FooBarWidget Thank You Hongli Lai, i will connect with @Theaxiom :)

**Comment by optimisticanshul** [comment]
@Theaxiom drop you an email 🎉

**Comment by Theaxiom** [comment]
Time to get to work. :)

**Comment by optimisticanshul** [comment]
@Theaxiom Any chance you got time to look at email :)

**Comment by Theaxiom** [comment]
@optimisticanshul I just replied to it. :)
